mapswipe_workers/mapswipe_workers/project_types/base/project.py

```diff
@@ -121,8 +121,13 @@ def save_project(self):
             del group["tasks"]
             groups[group["groupId"]] = group
         del project["groups"]
+
         project.pop("inputGeometries", None)
         project.pop("validInputGeometries", None)
+        project.pop("inputType", None)
```

Comment:
I think I overlooked this before. Why do you delete these project attributes? I think we can just keep them. They might be useful information that the people setting up projects can use later, e.g. to find out which projects they created already for which tasking manager projects

Reply:
keeping them now :+1: